nic_operator/yaml/ofed-nic-cluster-policy.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: mellanox.com/v1alpha1
+kind: NicClusterPolicy
+metadata:
+  name: nic-cluster-policy
+  namespace: mlnx-network-operator
+spec:
+  ofedDriver:
+    image: ofed-driver
+    repository: mellanox
+    version: 5.0-2.1.8.0
```

Comment:
it better to rander the yaml instead of putting hardcoded version

Reply:
did you address this comment?

Reply:
@moshe010 , yes see https://github.com/Mellanox/kubernetes-ci/compare/d4e093631ef0bb5bd41f2acc2d3b19cdbac3ab89..4a4d58b2bbcdc9ae19b81f5b7f13acffa634fff4